### PR TITLE
PR #23: Implementation for issue #64

### LINKED ISSUE #64: Trjectory problem with Case 1 and 2
At the moment, the trajectory of Case 1 and 2 (JSON files) of old AC3R do not provide sufficient points for the turn and right action, so that AC3RPlus generates strange lines which prevents a crash to occur. 

Suggestion fix from AC3R:
1. Use Ai Script to collect points from driving vehicles, make the scenario crash and manually modify the JSON files

Another approach:
1. Check the case whether Forward Impact
2. Check the straight lines
3. Add some points: a) Based on % b) Based on radius c) Add an extra straight line.

### PR DESCRIPTION
**Pull request by harveyvn**
Provide fixes for trajectory issues in Case 1, 2 and 3.

Screencasts:
Case 0 - 6 (except 3): https://www.dropbox.com/s/4bv60ur8b6m6uk0/C0-C6.mov?dl=0
Case 3: https://www.dropbox.com/s/i15twwxmjdt448u/C3.mov?dl=0

### CODE REVIEW

File: python_src/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,80 @@
-beamngpy~=1.18
-matplotlib~=3.3.2
-click~=7.1.2
-numpy~=1.19.2
-shapely~=1.7.1
-scipy~=1.5.2
-descartes~=1.1.0
-deap~=1.3.1
\ No newline at end of file
+appnope @ file:///opt/concourse/worker/volumes/live/6ca6f098-d773-4461-5c91-a24a17435bda/volume/appnope_1606859448531/work
```
Comment (alessiogambi) [inline python_src/requirements.txt]: This file contains references to your local installation and shall not be updated. Please revert the requirements.txt file to its previous version (restore)

**Review by alessiogambi — CHANGES_REQUESTED** [review]
Please, restore the requirements.txt file and we are ready to go

File: python_src/requirements.txt (hunk 1 above)
Comment (harveyvn) [reply python_src/requirements.txt]: Hi Dr. @alessiogambi, I brought back the requirements.txt file and added the separated enviroment.yml file for the Conda environment.

**Review by alessiogambi — APPROVED** [review]
LGTM

**Comment by harveyvn** [comment]
Hi Dr. @alessiogambi, due to the difference in the scale of horizontal and vertical axes in the chart, the length between points in Case 1, 2, and 3 looks different. I have rechecked, and all distances are about 5.

**Comment by alessiogambi** [comment]
Uhm... Can you check whether the option that sets the axis ratio to "equal" is activated?

**Comment by harveyvn** [comment]
> Uhm... Can you check whether the option that sets the axis ratio to "equal" is activated?

I set the axis ratio to equal, and here is a result.

image

image

**Comment by alessiogambi** [comment]
Those look exactly as I expected. And I suggest we always use this configuration in our plots because in reality there's no point in representing space/distances using axis with different scales.

**Comment by alessiogambi** [comment]
LGTM. We can merge this

**Pull request merged**